README.md

```diff
@@ -1,2 +1,48 @@
 # megafuel-go-sdk
-Golang SDK for the eoa-paymaster clients
+
+This is Golang SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
```

Comment:
==》`This Golang SDK is thin wrapper of MegaFuel clients, offering a streamlined interface to interact with the [MegaFuel](https://docs.nodereal.io/docs/megafuel-overview).`